lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -1858,12 +1857,7 @@ namespace Js
         library->AddFunctionToLibraryObject(typedarrayPrototype, PropertyIds::keys, &TypedArrayBase::EntryInfo::Keys, 0);
         JavascriptFunction* valuesFunc = library->AddFunctionToLibraryObject(typedarrayPrototype, PropertyIds::values, &TypedArrayBase::EntryInfo::Values, 0);
         library->AddMember(typedarrayPrototype, PropertyIds::_symbolIterator, valuesFunc);
-
-        if (scriptContext->GetConfig()->IsES7BuiltinsEnabled())
-        {
-            library->AddFunctionToLibraryObject(typedarrayPrototype, PropertyIds::includes, &TypedArrayBase::EntryInfo::Includes, 1);
-        }
-
+        library->AddFunctionToLibraryObject(typedarrayPrototype, PropertyIds::includes, &TypedArrayBase::EntryInfo::Includes, 1);
```

Comment:
I think it would be good to have tests for function length and the property attributes. Test262 might have these, but I'm not sure. If it does, they might be enough :wink: